tox.ini

```diff
@@ -160,18 +160,11 @@ commands =
 
 [testenv:package]
 skip_install = true
-# TODO(John Sirois): Include subprocess32 when #415, #658 and #694 are resolved.
+basepython = python3
```

Comment:
NB: We now can actually include it and things work properly when the pex is run with pypy or python2.7 or python3; however the overly verbose comment in `build_pex_pex` trumps.